Formula/adb-enhanced.rb

```diff
@@ -0,0 +1,38 @@
+# NOTE: Whenever making changes, here are some good ways to test changes
+# to this
+# 1. brew audit --strict --new-formula --online adb-enhanced
+# 2. brew install --verbose --build-bottle adb-enhanced
+# 3. brew install --include-test adb-enhanced && brew link adb-enhanced && brew test --verbose adb-enhanced
```

Comment:
```suggestion
```

we dont need this

Reply:
> we dont need this

What's the harm?
I am just leaving a comment that would be useful for me and anyone else in the future, who is planning to modify this formula

Reply:
it should be part of formula cookbook, again, we dont need this.

Reply:
OK. Removed.